Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.17
+
+WORKDIR /go/src/temporalite
+ADD . /go/src/temporalite/
+RUN go build -o /go/bin/temporalite /go/src/temporalite/cmd/temporalite
+
+VOLUME ["/temporalite"]
+
+EXPOSE 7233
+ENTRYPOINT ["/go/bin/temporalite", "start", "-f", "/temporalite/temporalite.db", "-n", "default", "-a" , "0.0.0.0"]
```

Comment:
Let's move this into a separate PR. I'd also love to use a multi-stage build to remove the go compiler dependencies from the resulting image.

Reply:
given my branch name I will keep this one for docker and submit another for frontend-ip-address.

Reply:
This PR is now for the Dockerfile only.

Based on distress guidance I have converted to a multistage build.